mono/dodge_the_creeps/Player.cs

```diff
@@ -3,86 +3,63 @@
 public partial class Player : Area2D
 {
     [Signal]
-    public delegate void Hit();
+    public delegate void HitEventHandler();
 
-    [Export]
-    public int speed = 400; // How fast the player will move (pixels/sec).
+    [Export] public int Speed { get; set; } = 400; // How fast the player will move (pixels/sec).
 
-    public Vector2 screenSize; // Size of the game window.
+    public Vector2 ScreenSize; // Size of the game window.
```

Comment:
Another public field that should be a property.

```suggestion
    public Vector2 ScreenSize { get; set; } // Size of the game window.
```